packages/waku/src/lib/builder/build.ts

```diff
@@ -252,6 +252,9 @@ const buildServerBundle = async (
     esbuild: {
       jsx: 'automatic',
     },
+    resolve: {
+      conditions: ['import', 'module', 'default', 'production'],
```

Comment:
Shouldn't we remove 'production'? Isn't it automatically added?

Reply:
Im not sure to be honest

Reply:
https://github.com/vitejs/vite/blob/6f030ec15f25a2a1d7d912f1b84d83ebb28a3515/packages/vite/src/node/plugins/resolve.ts#L1123-L1140

seems to work anyway. Let's remove `'production'` from our code.